design-spec/priviledge-decentralized-swupdates.tex

```diff
@@ -193,8 +245,92 @@ \section{Satisfying the requirements}
 
 % How does the different aspects of the design satisfies the requirements.
 
-% It seems that here we'd want to add the same subsections as in the
-% requirements section.
+\subsection{Open}
+\label{sec:sat-open}
+
+This design does not address the definition of minimal requirements for proposal
+submissions that will prevent spam attacks.
+
+This design addresses the requirement that any participant that can submit a
+common transaction should be able to vote on update proposals, by including
+defining a new \emph{vote} transaction type. Furthermore, the fee that these
+vote-transactions pay are calculated in the same way as monetary transactions.
```

Comment:
I think there is a misconception here. We say that votes and proposal submissions are fee-based transactions but the appropriate fee scheme for this purpose is left to be defined through proper incentives analysis (i.e., future work).
So it is not correct to say that we calculate fees for update transactions in the same way we calculate them for common transactions.

Reply:
Ok, I'll remove that sentence.

Reply:
However how can you say that the protocol will be open if we have no idea how much fee update transactions will pay?